include/engine/datafacade/datafacade_base.hpp

```diff
@@ -162,6 +162,8 @@ class BaseDataFacade
 
     virtual bool GetContinueStraightDefault() const = 0;
 
+    virtual bool UseLeftSideDriving() const = 0;
```

Comment:
`MockDataFacade` needs a dummy impl. for this, too. Otherwise unit tests won't compile.